core/frontend/src/tools/ToolAssistance.ts

```diff
@@ -64,6 +64,48 @@ export enum ToolAssistanceInputMethod {
   Touch,
 }
 
+/** Common Tool Assistance main instruction prompt messages
+ * @public
+ * @extensions
+ */
+export enum ToolAssistancePromptKey {
```

Comment:
I prefer to use const members of an “as const” object over string enums. It’s more efficient unless you plan to do reverse lookups.

Reply:
I'll switch it around.

Reply:
Changed.